build.gradle.kts

```diff
@@ -1,20 +1,13 @@
-import com.adarshr.gradle.testlogger.theme.ThemeType
-import org.jetbrains.intellij.tasks.RunPluginVerifierTask.VerificationReportsFormats.*
-import org.jetbrains.intellij.tasks.RunPluginVerifierTask.FailureLevel.*
-
 fun properties(key: String) = providers.gradleProperty(key)
 fun environment(key: String) = providers.environmentVariable(key)
 fun prop(name: String): String {
     return properties(name).get()
 }
+
 plugins {
-    java // Java support
-    alias(libs.plugins.kotlin) // Kotlin support
-    alias(libs.plugins.gradleIntelliJPlugin) // Gradle IntelliJ Plugin
-    alias(libs.plugins.changelog) // Gradle Changelog Plugin
```

Comment:
None of these were brought over yet, though I know they're needed for the build tasks below.